src/main/java/org/team1540/elmo/Constants.java

```diff
@@ -40,4 +40,16 @@ public final class Constants {
     public static final int DEPLOY_INNER_CHANNEL = 6;
     // COMPRESSOR
     public static final int COMPRESSOR = 0;
+
+    // INPUT
+    public static final int NAVX_PORT = 0;
```

Comment:
When you declare the NavX later, you don't actually use this constant, so it might be clearer to I just remove this

Reply:
Yep yes yeah